build-system/tasks/e2e/network-logger.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {logging} from 'selenium-webdriver';
+
+/** @enum {string} */
+const NetworkMethods = {
+  'REQUEST_WILL_BE_SENT': 'Network.requestWillBeSent',
```

Comment:
Are there other network methods that might be worth keeping track of in future? If so, would it make sense to list the rest of the values here?

Reply:
Sure thing, added a few more. I didn't write an API for them because I don't know what part of the message would be useful to filter on - something we can add as needed.